presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoInformer.java

```diff
@@ -23,32 +23,40 @@
 
 import java.io.IOException;
 import java.nio.file.Path;
-import java.nio.file.Paths;
 
 import static com.teradata.prestomanager.agent.AgentFileUtils.getFileProperty;
 import static java.lang.String.format;
 import static javax.ws.rs.core.MediaType.TEXT_PLAIN;
 import static javax.ws.rs.core.UriBuilder.fromUri;
 
-public class PrestoUtils
+/**
+ * Retrieves installation-independent information about the Presto installation
```

Comment:
It's information about Presto that is independent of the details of the Presto installation.

Though I guess that's not true any more, so I'll remove that.